src/components/backend-ai-user-dropdown-menu.ts

```diff
@@ -586,76 +402,15 @@ export default class BackendAiUserDropdownMenu extends LitElement {
-            label="${_t('general.SecretKey')}"
-            style="margin-bottom:20px; margin-top:20px;" value="" readonly>
-        </mwc-textfield>
-      </div>
-      <div slot="content" class="layout vertical" style="width:300px;">
-        <mwc-textfield id="pref-original-password" type="password"
-            label="${_t('webui.menu.OriginalPassword')}" maxLength="64"
-            style="margin-bottom:20px;">
-        </mwc-textfield>
-        <div class="horizontal flex layout">
-          <mwc-textfield id="pref-new-password" label="${_t('webui.menu.NewPassword')}"
-              type="password" maxLength="64"
-              auto-validate validationMessage="${_t('webui.menu.InvalidPasswordMessage')}"
-              pattern=${BackendAiCommonUtils.passwordRegex}
-              @change="${this._validatePassword}">
-          </mwc-textfield>
-          <mwc-icon-button-toggle off onIcon="visibility" offIcon="visibility_off"
-                                    @click="${(e) => this._togglePasswordVisibility(e.target)}">
-          </mwc-icon-button-toggle>
-        </div>
-        <div class="horizontal flex layout">
-          <mwc-textfield id="pref-new-password2" label="${_t('webui.menu.NewPasswordAgain')}"
-              type="password" maxLength="64"
-              @change="${this._validatePassword}">
-          </mwc-textfield>
-          <mwc-icon-button-toggle off onIcon="visibility" offIcon="visibility_off"
-                                    @click="${(e) => this._togglePasswordVisibility(e.target)}">
-          </mwc-icon-button-toggle>
-        </div>
-        ${this.totpSupported ? html`
-          <div class="horizontal flex layout">
-            <p style="flex-grow: 1;margin-left: 15px;">
-              ${_t('webui.menu.TotpActivated')}
-            </p>
-            <mwc-switch id="totp-activation" .selected="${this.totpActivated}" style="margin-right: 10px;"
-                @click="${(e) => this.totpActivated ? this._confirmRemovingTotp(e) : this._startActivatingTotp(e)}">
-            </mwc-switch>
-          </div>
-        `: html``}
-      </div>
-      <div slot="footer" class="horizontal end-justified flex layout">
-        <div class="flex"></div>
-        <mwc-button
-            label="${_t('webui.menu.Cancel')}"
-            @click="${this._hideUserPrefDialog}"></mwc-button>
-        <mwc-button
-            unelevated
-            label="${_t('webui.menu.Update')}"
-            @click="${this._updateUserInformation}"></mwc-button>
-      </div>
-    </backend-ai-dialog>
+    <backend-ai-react-user-profile-dialog
+      value="${JSON.stringify({isOpen: this.isOpen})}"
+      @cancel="${()=> this._hideUserPrefDialog()}"
+      @refresh="${()=> this._refreshUserInfoPanel()}"
+      @updateFullName="${(e)=> this._updateUserFullName(e.detail.newFullName)}"
+      @confirmRemovingTotp="${()=> this._confirmRemovingTotp()}"
+      @startActivatingTotp="${()=> this._startActivatingTotp()}"
```

Comment:
I think it's better to handle them in the `UserProfileSettingModal`.

Reply:
@Sujin-Kim1 
I removed @refresh props according to your review, because that was unnecessary props.
But other props necessary!
@updateFullName props is need to change user full name in header after mutation
@confirmRemovingTotp and @startActivatingTotp is need to open other modal and close user profile modal.

Reply:
@Sujin-Kim1 
I tried to use "current-user-info-changed", but the full name side by dropdown button was not changed after user changed info.
I think must to use @updateFullName or @refresh😔